inc/admin/options-page.php

```diff
@@ -89,6 +89,18 @@ public function register_settings_and_fields() {
 		);
 
 		add_settings_field(
+			'email_url',
+			__( 'Email', 'alcatraz' ),
+			array( $this, 'field_text' ),
+			'alcatraz_settings_section',
+			'social_media_links',
+			array(
+				'id'          => 'email_url',
+				'description' => __( 'Enter your email address (mailto:me@example.com)', 'alcatraz' ),
```

Comment:
I think we should allow users to enter plain emails like carrie@carrie.com and wrap this in the mailto: on the PHP side rather than have users enter the full href here.